docs/firstobject.md

```diff
@@ -76,78 +76,81 @@ There is of course a lot more to Nodes, Content Types, Formatters, Widgets and i
+![Ingest Step 4 - Attach Files pt 1](../imgs/04_ingest-step-four_upload-files_1.png)
 
-Two things to note: The `Save Metadata` button simply persists the metadata in the current webform session. The actual ingest of the Object happens when you click `Save` (see the **Complete** step below). Secondly, each "upload field" has a set of allowed file types (such as .jpg, .tiff, .png) and in this demo there are also file size restrictions which are explained in the Upload Field's description.
+Once you've uploaded your file, you will see all the Exif data extracted from the image.
 
-From here, Steps 5 & 6 (hotspots & Preview) are skipped and you are directed to the **Complete** step.
+![Ingest Step 4 - Attach Files pt 2](../imgs/04_ingest-step-four_upload-files_2.png)
 
-### Complete
+Once you've mentally digested all of that data, let's go ahead and click `Save Metadata`.
+
+**Question:** "Wait! Don't I need to switch `Save as` from *Draft* to *Publish*? I'm ready for this Digital Object to be shared with the world!"
 
-![Ingest Step7](../imgs/ingest-step-7_Complete.jpg)
+**Answer:** You can, but it's not necessary to switch to *Publish* now. By clicking `Save Metadata` we are simply persisting the metadata in the current webform session. So, we can confidently leave the status as *Draft* as we click `Save Metadata` because all we're doing here is... well, *saving* all the *metadata* we've entered so far.
 
-You made it! You added your Metadata, Linked Data, Uploaded your files and now you're ready to save! First, give your object a preview by clicking the `Preview` button directly to the right of the `Save` button.
+### Complete
+Alright, we've made it. We've added metadata, linked Data, uploaded our files and now... we're ready to save! Go ahead and change the status from *Draft* to *Published* and click `Save`.
 
-Does everything look alright? Great! Click the `Back to content editing` button in the top left corner to go through with saving your object.
+![Ingest Step 7 - Complete 1](../imgs/06_ingest_step-six_complete.png)
 
-![Preview-page-screenshot](../imgs/ingest-documentation_preview-page.png)
+Once you hit save you should see the following green messages and your first Archipelago Digital Object!
 
-Now you are back to where you began, and you are ready to save your First Digital Object! **Be sure to switch your "Save As" status from *Draft* to *Publish*.** Once you hit `Save` you should see a few green push notifications indicating your object has been created. 
+![Ingest Step 7 - Complete Green Messages](../imgs/06_ingest-step-six_complete_green-messages.png)
 
-Like this:
-![Ingest Step7 Saved](../imgs/ingest-step-7_Complete-save.jpg)
+![Ingest Step 7 - Complete 2](../imgs/06_ingest-step-six_complete_2.png)
 
-Congratulations on creating your first digital object!
```

Comment:
C'mon. Let's keep the encouragement, don't you like to congratulate people for following your guide? 😄